frontend/src/pages/journal/libs/constants/constants.ts

```diff
@@ -0,0 +1,8 @@
+const DEFAULT_NOTE_PAYLOAD = {
+  title: 'New note',
+  text: '{"root":{"children":[{"children":[{"detail":0,"format":0,"mode":"normal","style":"","text":"","type":"text","version":1}],"direction":"ltr","format":"","indent":0,"type":"paragraph","version":1}],"direction":"ltr","format":"","indent":0,"type":"root","version":1}}',
```

Comment:
What is a purpose of it's text's default value

Reply:
the lexical editor state needs to accept this structure for the initial value. If we would pass just text there would be an error. You can say, but we pass the title as the normal text. Yes, but we also send it as a simple text to the backend. By setting the text with simple text we would need to make a conversion like with title so that lexical would accept this data on the initial render. Then we would need to check whether this is a simple text from default values or actually a lexical structure from the backend which I think is an unnecessary operation. Please tell me if you have more questions)

Reply:
I think the backend shouldn't know anything about our approach, btw it's strange to have different methods of conversioning. It had better save simple string, without json structure related to that lib

Reply:
first, the backend accepts it just as simple string text, I don't think this is a problem. But the main thing is that with converting to the simple text we will make the whole library unnecessary and remove all its features as new lines and so on.  Furthermore, I am pretty sure that we would need this structure so that lexical would make a rich text. I am talking about the rich text because we have an updated design for the journal which implies the use of it

Reply:
Are you sure that mobile side will handle it as exppected?

Reply:
about title. We need to convert to simple text because we have a restriction for title length which does not let to pass lexical structure. That is why we needed to convert it to simple text. And of course, I don't think that title should have some rich text features

Reply:
I am just wondering about different approaches for title and text, it is confusing

Reply:
if there would the problems with mobile, we can convert it to HTML then, and send it to the backend

Reply:
Yep, let do it)

Reply:
> I am just wondering about different approaches for title and text, it is confusing

I see. But let's assume we need to make the words bold, italic, etc. We wouldn't have this opportunity with simple text, but with lexical structure it is possible

Reply:
> Yep, let do it)

ok, then I will handle it

Reply:
Good point, but it's overhead for this time)